Automattic/wp-calypso
# Refactors GSuiteUserItem

#### Changes proposed in this Pull Request

* Moves sass into js import
* Makes component a functional component
* Switch to useTranslate hook
* Adds tests
* Renames to G Suite over GoogleApps

#### Testing instructions

* Have site with G Suite
* Goto site
* Click domains in left menu
* Click Email tab
* Observe user row renders correctly
* Click link and observe that it goes out to Google login

<img width="1064" alt="Screen Shot 2019-03-20 at 11 37 58 PM" src="https://user-images.githubusercontent.com/6817400/54732295-6c2c4280-4b69-11e9-8871-77520c7e5516.png">
